dashboard/app/helpers/ai_system_prompts/evaluate_system_prompt_helper.rb

```diff
@@ -1,18 +1,23 @@
 module AiSystemPrompts::EvaluateSystemPromptHelper
   def self.get_system_prompt(level, evaluation_type)
+    additional_directions = level.respond_to?(:additional_ai_evaluation_instructions) && level.additional_ai_evaluation_instructions.present? ? level.additional_ai_evaluation_instructions.strip + "\n" : ''
     evaluation_criteria = get_evaluation_criteria(level)
+    task_context = "Your task is to review the student's work."
+    if additional_directions.present?
+      task_context << " #{additional_directions}"
+    end
```

Comment:
I think it could be good to add a quick test to confirm this shows up as expected https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/test/helpers/system_prompt_helper_test.rb